### DISCUSSION TITLE
Robust version of Multiple Regression

### DISCUSSION THREAD

**Discussion by freudrage** [raphaelvallat/pingouin#248] [Q&A] [answered]
Hi, is there a way to do a robust version of multiple regression in pingouin? I'm having trouble locating a method for robust multiple regression in a number of stats packages. Everything seems to be for one predictor variable. I am a psychology student, new to python and regression analyses , though -- any thoughts?

**Comment by raphaelvallat** [comment, marked as answer]
Hi @freudrage,

You just opened the very first GitHub Discussions, so thank you :-) Robust linear regression is not implemented in Pingouin, however, it is implemented in statsmodels ([RLM](https://www.statsmodels.org/stable/generated/statsmodels.robust.robust_linear_model.RLM.html#statsmodels.robust.robust_linear_model.RLM)):

Thanks,
Raphael

**Reply by freudrage** [reply]
Hi Raphael, ok thank you so much.
